src/extension/rn-extension.ts

```diff
@@ -49,6 +50,10 @@ export function activate(context: vscode.ExtensionContext): void {
                     const nodeDebugPath = vscode.extensions.getExtension("andreweinand.node-debug").extensionPath;
                     const fsUtil = new FileSystem();
                     fsUtil.writeFile(path.resolve(__dirname, "../", "debugger", "nodeDebugLocation.json"), JSON.stringify({ nodeDebugPath })).done();
+                }).then(() => {
+                    let server = new ExtensionServer();
+                    server.setup();
```

Comment:
I see that `setup` returns a promise, but you don't care about the return value here. Should we be waiting until the server configures successfully or otherwise informing the user that something has gone wrong?